mlflow/gateway/providers/mistral.py

```diff
@@ -54,6 +54,36 @@ def model_to_completions(cls, resp, config):
             ),
         )
 
+    @classmethod
+    def model_to_chat(cls, resp, config):
```

Comment:
note: We only have completion endpoint support for Mistral now. However, their endpoint is chat fornat (OpenAI compatible) .